### PR TITLE
fix(rust):replaced termimad with formatdoc & opts.terminal.stdout() in authenticated

### LINKED ISSUE #6678: Build output of `authenticated` commands implementing the `Output` trait instead of using `termimad`
## Current behavior

The [`authenticated` commands](https://github.com/build-trust/ockam/blob/02f62ab5fdc9ab7508c5a148a8f9d97325acc92c/implementations/rust/ockam/ockam_command/src/authenticated.rs#L64) uses `termimad` to format and print its output.

## Desired behavior

The command output should be formatted with `formatdoc` and printed with `opts.terminal.stdout()`.

## Implementation details

The `get` and `list` commands are using the function `print_entries` to build the output, which uses a list of `identifier, attribute` pairs. Consider creating a custom struct to represent that pair and implement the `Output` trait for it so that you can build the output as follows:

```rust
opts.terminal
    .stdout()
    .plain(&attribute) // use `opts.terminal.build_list(&attributes, ...)` to build the output for the `list` command
    .json(serde_json::json!(&attribute))
    .write_line()?;
```

Check the [`lease list` command](https://github.com/build-trust/ockam/blob/da8e3a5e109ed2ed265cdbac05a9f9e890e21451/implementations/rust/ockam/ockam_command/src/lease/list.rs#L57) to see an example of how to do this.

---

<i>

Some tips for newcomers:
1. Create a new branch for the issue you are working on (don't push directly to your fork's `develop` branch)
1. Run `cargo fmt`. You can configure the editor to run it automatically everytime a file is saved
1. Run `cargo clippy` to make sure the code compiles + follows rust standards/best practices
1. You don't need to keep your branch updated with `develop` until there is a merge conflict. If you still want to update it, use `git rebase develop` instead of `gir merge develop` 

We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).

</i>

### PR DESCRIPTION
**Pull request by zacharyftw**
## Current behavior
The [authenticated commands](https://github.com/build-trust/ockam/blob/02f62ab5fdc9ab7508c5a148a8f9d97325acc92c/implementations/rust/ockam/ockam_command/src/authenticated.rs#L64) uses termimad to format and print its output.

## Proposed changes
The command output has been formatted with formatdoc and printed with opts.terminal.stdout().

fixes https://github.com/build-trust/ockam/issues/6678

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] There are no Merge commits in this Pull Request. Ockam repo maintains a linear commit history. We merge Pull Requests by rebasing them onto the develop branch. Rebasing to the latest develop branch and force pushing to your Pull Request branch is okay.
- [x] I have read and accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have read and accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/.github/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a separate Pull Request, this will mark the commit as verified.

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/authenticated.rs [hunk 1]
```diff
@@ -95,45 +84,49 @@ async fn make_background_node_client(
...
-                format!("{:?}", entry.added()),
-                entry
-                    .expires()
-                    .map_or("-".to_string(), |t| format!("{:?}", t)),
-                entry.attested_by().map_or("-".to_string(), String::from),
-            )
-        })
-        .collect();
-    let mut expander = template.expander();
+struct IdentityEntry<'a> {
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/src/authenticated.rs]: We could name this `IdentifierWithAttribues`.

File: implementations/rust/ockam/ockam_command/src/authenticated.rs [hunk 2]
```diff
@@ -95,45 +84,49 @@ async fn make_background_node_client(
...
-                .set("identifier", identifier)
-                .set("attributes", attributes)
-                .set("created_at", created_at)
-                .set("expires_at", expires_at)
-                .set("attested_by", attested_by);
-        },
-    );
-    let skin = MadSkin::default();
-    skin.print_expander(expander);
+impl<'a> Display for IdentityEntry<'a> {
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_command/src/authenticated.rs]: We want to implement the `Output` trait here, so you can print it like in the example from the issue description: https://github.com/build-trust/ockam/blob/da8e3a5e109ed2ed265cdbac05a9f9e890e21451/implementations/rust/ockam/ockam_command/src/lease/list.rs#L57

**Review by adrianbenavides — COMMENTED** [review]
Hi @KekmaTime , thanks for submitting this PR. I've left some comments, let me know if you have any question 👍

**Comment by mrinalwadhwa** [comment]
@KekmaTime while we still have to review this change. I'm adding the `HACKTOBERFEST-ACCEPTED` label so the contribution is counted by hacktoberfest bots before the end of October. Thank you for taking time to contribute 🎉 

Please note @adrianbenavides's feedback above.

**Comment by zacharyftw** [comment]
@adrianbenavides is this okay?

**Review by etorreborre — APPROVED** [review]

**Pull request merged**